keyboards/keebio/bfo9000/keymaps/vial/rules.mk

```
@@ -0,0 +1,6 @@
+VIA_ENABLE = yes
+VIAL_ENABLE = yes
+LTO_ENABLE = yes
+TAP_DANCE_ENABLE = no
+COMBO_ENABLE = no

```

```suggestion
COMMAND_ENABLE = no
```
If you disable `COMMAND` ([which the QMK docs suggest](https://docs.qmk.fm/#/feature_command),  since you can use magic keycodes instead), you can keep these other features with about 1.6kb to spare.